api/app/Rules/HasLanguageRequirements.php

```diff
@@ -35,7 +35,7 @@ public function passes($attribute, $value)
         $userLookingForBilingual = $thisUser->looking_for_bilingual;
         $poolNeedsBilingual = $this->pool->advertisement_language == PoolLanguage::BILINGUAL_INTERMEDIATE->name || $this->pool->advertisement_language == PoolLanguage::BILINGUAL_ADVANCED->name;
 
-        $passes = ! $poolNeedsBilingual || ($poolNeedsBilingual && $userLookingForBilingual);
+        $passes = ! $poolNeedsBilingual || $userLookingForBilingual;
```

Comment:
While logically this is correct, I think it makes it harder to understand.  I can easily understand the business rules of the first form: "Either the pool doesn't need bilingual or the pool does need bilingual and the user is also bilingual".  But reading the shorter form, I can't easily see what it's trying to express.  There is an extra mental jump since the fact the the pool does need bilingual is just implied on the right side of the OR.

Reply:
Hmm, we might need a third person to come in and break the tie because I found the exact opposite. The double check of `$poolNeedsBilingual` made it more confusing for me which was why I was so uncertain about this change :sweat_smile:

Reply:
Yep, we can get some other opinions. :+1:   Another options would be to assign the second part of the expression to its own variable to make it clearer.  Something like

```php
$userLookingForBilingual = $thisUser->looking_for_bilingual;
$poolNeedsBilingual = $this->pool->advertisement_language == PoolLanguage::BILINGUAL_INTERMEDIATE->name || $this->pool->advertisement_language == PoolLanguage::BILINGUAL_ADVANCED->name;
$poolAndUserAreBothBilingual = $poolNeedsBilingual && $userLookingForBilingual;

$passes = ! $poolNeedsBilingual || $poolAndUserAreBothBilingual;
```